tests/test_bundle.py

```diff
@@ -258,20 +257,31 @@ def _test_bundle_put(self, replica: Replica, fixtures_bucket: str):
             requests.codes.conflict,
         )
 
-        # should *NOT* be able to upload a bundle with a missing file, but we should get requests.codes.conflict.
+        print('should *NOT* be abke to this without bundle version.')
```

Comment:
Spelling.

This test used to be addressing `requests.codes.conflict`. Has that changed?